pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1000,6 +1000,11 @@ func (m *kubeGenericRuntimeManager) computePodActions(ctx context.Context, pod *
 			reason = reasonStartupProbe
 		} else if isInPlacePodVerticalScalingAllowed(pod) && !m.computePodResizeAction(pod, idx, containerStatus, &changes) {
 			// computePodResizeAction updates 'changes' if resize policy requires restarting this container
+			// overcome slow reconcile ( range of 1-3 minutes ) when reverting resize patch quickly,
+			// before container is started from previous resize patch
+			// it will still wait for kubelet's next reconcile loop (interval is 1 minute), issue 123940
```

Comment:
nit: I don't think you need to reference the issue here. If someone wants, they can dig it up from git blame.
```suggestion
			// Resize restart should be effective immediately and clear any accumulated backoff.
```

Reply:
Done